### PR TITLE
[FEATURE] Automatic inclusion of PageTsConfig of extensions

### PR DESCRIPTION
**Pull request by linawolf**
https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Feature-96614-AutomaticInclusionOfPageTsConfigOfExtensions.html

refs https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1624

### CODE REVIEW

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 1]
```diff
@@ -31,60 +31,47 @@ this is not recommended anymore.
 Setting the Page TSconfig globally
 ==================================
 
-Many page TSconfig settings can be set globally. This is useful for
-installations that contain only one site and use only one sitepackage extension.
+.. versionadded:: 10.2
```
Comment (brotkrueml) [inline Documentation/UsingSetting/PageTSconfig.rst:34]: I assume this should be 12.0, not 10.2?

Comment (linawolf) [reply Documentation/UsingSetting/PageTSconfig.rst]: Of course. fixed

**Comment by linawolf** [comment]
@brotkrueml I resolved your comment, anything else or can the PR be merged like this?

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 2]
```diff
@@ -31,60 +31,47 @@ this is not recommended anymore.
...
+.. versionadded:: 12.0
+   Starting with TYPO3 12.0 page TSconfig in a file named
+   :file:`Configuration/page.tsconfig` in an extension is automatically
+   loaded during build time.
 
-Extensions supplying custom default Page TSconfig that should always be included,
-can also set the Page TSconfig globally.
+Global page TSconfig should be stored within an extension, usually a sitepackage
+extension. The content of the file :file:`Configuration/page.tsconfig` within
+and extension is automatically loaded during build time.
```
Comment (susannemoog) [inline Documentation/UsingSetting/PageTSconfig.rst:41]: an instead of and

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 3]
```diff
@@ -31,60 +31,47 @@ this is not recommended anymore.
...
 
-There is a global `TYPO3_CONF_VARS` value called
-:ref:`$GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPageTSconfig'] <t3coreapi:typo3ConfVars_be_defaultPageTSconfig>`.
+Many page TSconfig settings can be set globally. This is useful for
+installations that contain only one site and use only one sitepackage extension.
 
-The API function above adds content to that array. The array value itself
-however should **not** be changed or set directly (for example in the
-:file:`LocalConfiguration.php`).
+Extensions supplying custom default Page TSconfig that should always be included,
```
Comment (susannemoog) [inline Documentation/UsingSetting/PageTSconfig.rst:56]: do we write "page TSconfig" or "Page TSconfig" ?

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 4]
```diff
@@ -31,60 +31,47 @@ this is not recommended anymore.
...
-Page TSconfig in a project-specific
-:doc:`sitepackage <t3sitepackage:Index>` extension.
+Global page TSconfig, compatible with TYPO3 11 and 12
+-----------------------------------------------------
 
-Use the :typoscript:`@import '...'` syntax to keep the Page TSconfig in a
-separate file.
+In TYPO3 11 installations the content of file:`Configuration/page.tsconfig`
+is not loaded automatically yet. You can achive compatibility with both
+TYPO3 11 and 12 by importing the content of this file with the API function
```
Comment (susannemoog) [inline Documentation/UsingSetting/PageTSconfig.rst:64]: is it then imported twice in v12 // does this need a version condition surrounding it?

**Review by susannemoog — CHANGES_REQUESTED** [review]

**Review by susannemoog — APPROVED** [review]

**Pull request merged**